interop-tests/Cargo.toml

```diff
@@ -10,10 +10,24 @@ anyhow = "1"
 either = "1.8.0"
 env_logger = "0.10.0"
 futures = "0.3.27"
-libp2p = { path = "../libp2p", features = ["websocket", "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros"] }
+libp2p = { path = "../libp2p", features = [
+    "websocket",
+    "yamux",
+    "tcp",
+    "tokio",
+    "ping",
+    "noise",
+    "tls",
+    "dns",
+    "rsa",
+    "macros"
+] }
 libp2p-quic = { path = "../transports/quic", features = ["tokio"] }
 libp2p-webrtc = { path = "../transports/webrtc", features = ["tokio"] }
+libp2p-mplex = { path = "../muxers/mplex" }
 log = "0.4"
 rand = "0.8.5"
-redis = { version = "0.22.1", default-features = false, features = ["tokio-comp"] }
+redis = { version = "0.22.1", default-features = false, features = [
+    "tokio-comp"
+] }
```

Comment:
Let's keep the diff minimal and focused on the change at hand please.

Reply:
@thomaseizinger Sure, I'll remove this if you disagree, this is just my settings for `.toml` files, using prettier extension. I thought it was good to correct the small formatting stuffs as long as I can but I'll remove it, no worries.

Reply:
> @thomaseizinger Sure, I'll remove this if you disagree, this is just my settings for `.toml` files, using prettier extension. I thought it was good to correct the small formatting stuffs as long as I can but I'll remove it, no worries.

I am all for consistent formatting but we should enforce it in CI too then. At the moment, I am not aware of a standard way across the Rust ecosystem to format toml files which is why we don't have something in place.

Feel free to send a separate PR with just formatting changes.